.github/CODEOWNERS

```diff
@@ -1,3 +1,4 @@
-### DevOps team should be notified if a modification on workflow is done
+### DevOps team is notified to garantee consistency ansd security good pratices
+### ci-owners team is notified to maintain and improve
 
-/.github/workflows/ @pass-culture/sherif-ops @pass-culture/sherif-devlead
+/.github/workflows/ @pass-culture/sherif-ops @pass-culture/ci-owners
```

Comment:
j'aimerais bien ne plus etre pingé pour les modifications du repos main

si on a un groupe séparé, c'est super chouette !

Reply:
@francois-seguin @mageoffray votre avis sur le fait de créer deux groupes, puisque deux repo gérés par deux équipes différentes ?
@lixxday a dit, il me semble, lors de la réu, qu'il préférait un seul groupe.

Reply:
On peut faire un second groupe `app-ci-owners` si le nombre plus restreint de membres coté tribe Jeunes n'est pas un obstacle.

Reply:
Deux groupes alors ? Nommons les alors pour les ~~lier~~désigner tous. Et coller aux noms des repos et que cela soit plus clair à comprendre.

- native-ci-owners
- main-ci-owners

Et si plus tard on scinde le repo main en pcapi et pro (cf chantier CD), on pourra scinder en deux groupes le groupe main-ci-owners.
Si ok pour tous, je modifie les PR.

Reply:
plutot `app-ci-owners`, le fait que ça doit du natif, me semble plus un détail technologique

(peu importe le nommage tant que c'est splitté, je suis content)

Reply:
je viens de pusher le nouveau nom de groupe, et j'ai separé en deux groupes.
Si ok pour toi @bruno-ebstein-pass-culture je squash and merge ?